src/github.com/mozilla-services/pushgo/simplepush/worker.go

```diff
@@ -789,5 +801,13 @@ func (r *NoWorker) Flush(_ *PushWS, lastAccessed int64, channel string, version
 	return nil
 }
 
+func cleanChannelIDs(ids []json.RawMessage) (clean []string) {
```

Comment:
This appears to remove the front and last character of every item. The function name isn't terribly descriptive, nor is it clear why they need to be 'cleaned'. A doc string would help a lot.